src/panels/config/cloud/dialog-cloud-already-connected/dialog-cloud-already-connected.ts

```diff
@@ -61,7 +61,27 @@ class DialogCloudAlreadyConnected extends LitElement {
             )}
           </b>
         </div>
-        <div class="instance-details">
+        ${details.name
+          ? html` <div class="instance-details">
+              <span>
+                ${this.hass.localize(
+                  "ui.panel.config.cloud.dialog_already_connected.instance_name"
+                )}:
+              </span>
+              <span>${details.name})}</span>
+            </div>`
+          : nothing}
+        ${details.version
+          ? html` <div class="instance-details">
+              <span>
+                ${this.hass.localize(
+                  "ui.panel.config.cloud.dialog_already_connected.instance_version"
+                )}:
+              </span>
+              <span>${details.version})}</span>
+            </div>`
+          : nothing}
+        <div class="instance-detail">
```

Comment:
```suggestion
        <div class="instance-details">
        ${details.name
          ? html` <div class="instance-detail">
              <span>
                ${this.hass.localize(
                  "ui.panel.config.cloud.dialog_already_connected.instance_name"
                )}:
              </span>
              <span>${details.name})}</span>
            </div>`
          : nothing}
        ${details.version
          ? html` <div class="instance-detail">
              <span>
                ${this.hass.localize(
                  "ui.panel.config.cloud.dialog_already_connected.instance_version"
                )}:
              </span>
              <span>${details.version})}</span>
            </div>`
          : nothing}
```
Ah the diff is weird but I don't think the div classes are correct. Should have a few `.instance-detail` inside one `.instance-details`